includes/classes/blocks/class-template-parts.php

```
@@ -0,0 +1,267 @@
+		$template_parts = apply_filters( 'lsx_to_template_parts', $template_parts );
+
+		$theme = get_stylesheet();
+
+		if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+			error_log( sprintf( 'Template Parts: Processing %d template parts for theme: %s', count( $template_parts ), $theme ) );
+		}
+
+		foreach ( $template_parts as $slug => $args ) {
+			$this->maybe_create_template_part( $slug, $args, $theme );
+		}
+	}
```

PHPMD (2.15.0)

The method create_template_parts is not named in camelCase. (undefined)

(CamelCaseMethodName)